src/cryptography/x509/extensions.py

```diff
@@ -1152,6 +1155,51 @@ def __hash__(self):
 
 
 @utils.register_interface(ExtensionType)
+class PrecertificateSignedCertificateTimestamps(object):
```

Comment:
If we're not going to implement __hash__ on this I suppose we need to add it to the list on that old issue.